$OpenBSD: patch-openscad_pro,v 1.3 2019/11/06 13:21:51 rsadowski Exp $

- remove unsupported compiler flag (that is NOT ignored)
- don't use bundled Liberation fonts, use RDEP on fonts/liberation-fonts instead
- fix manpage installation dir
- disable hidapi support, it doesn't compile

Index: openscad.pro
--- openscad.pro.orig
+++ openscad.pro
@@ -149,7 +149,6 @@ netbsd* {
 # See Dec 2011 OpenSCAD mailing list, re: CGAL/GCC bugs.
 *g++* {
   QMAKE_CXXFLAGS *= -fno-strict-aliasing
-  QMAKE_CXXFLAGS_WARN_ON += -Wno-unused-local-typedefs # ignored before 4.8
 
   # Disable attributes warnings on MSYS/MXE due to gcc bug spamming the logs: Issue #2771
   win* | CONFIG(mingw-cross-env)|CONFIG(mingw-cross-env-shared) {
@@ -192,7 +191,7 @@ CONFIG += lib3mf
 CONFIG += gettext
 CONFIG += libxml2
 CONFIG += libzip
-CONFIG += hidapi
+#CONFIG += hidapi
 CONFIG += spnav
 CONFIG += double-conversion
 
@@ -716,10 +715,6 @@ libraries.path = "$$PREFIX/share/$${FULLNAME}/librarie
 libraries.files = libraries/*
 INSTALLS += libraries
 
-fonts.path = "$$PREFIX/share/$${FULLNAME}/fonts/"
-fonts.files = fonts/*
-INSTALLS += fonts
-
 colorschemes.path = "$$PREFIX/share/$${FULLNAME}/color-schemes/"
 colorschemes.files = color-schemes/*
 INSTALLS += colorschemes
@@ -740,7 +735,7 @@ icons.path = $$PREFIX/share/pixmaps
 icons.extra = test -f icons/$${FULLNAME}.png && cp -f icons/$${FULLNAME}.png \"\$(INSTALL_ROOT)$${icons.path}/\" || cp -f icons/openscad.png \"\$(INSTALL_ROOT)$${icons.path}/$${FULLNAME}.png\"
 INSTALLS += icons
 
-man.path = $$PREFIX/share/man/man1
+man.path = $$PREFIX/man/man1
 man.extra = cp -f doc/openscad.1 \"\$(INSTALL_ROOT)$${man.path}/$${FULLNAME}.1\"
 INSTALLS += man
 
